moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -546,7 +546,6 @@ const ProjectFundingTable = () => {
                 objects: {
                   ...newData,
                   project_id: projectId,
-                  added_by: userId,
```

Comment:
The `graphql-engine` instance is configured by this PR to not accept any audit field values from insert mutation, because it figures out who did what based on the authentication token given to the endpoint. That's why this came out and nothing went back in. I appreciate you all considering this and thinking about if i'm looking at this right.

Reply:
Thanks for thinking that through with me!